File: GettingStarted/countDigits.cpp
#include <iostream>
using namespace std;

int countDigits(int n){
    int count=0;
    int num=n;
    while(num!=0){
        num/=10;
        count++;
    }
    return count;
}

int main(int argc, char **argv){
    int n;
    cin >> n;
    cout<<countDigits(n)<<endl;
    
}

File: GettingStarted/countDigits.java
import java.util.Scanner;

public class Main{
    public static Scanner scn = new Scanner(System.in);
    public static int countDigits(int n){
    int count=0;
    int num=n;
    while(num!=0){
        num/=10;
        count++;
    }
    return count;
}

public static void main(String[] args){
    int n = scn.nextInt();
    System.out.println(countDigits(n));
    
}
}